components/rmrk/Gallery/Search/SearchSortDropdown.vue

```diff
@@ -37,3 +38,11 @@ export default class SearchSortDropdown extends Vue {
   }
 }
 </script>
+
+<style scoped lang="scss">
+ .select-dropdown {
+  @media screen and (max-width: 1216px) and (min-width: 768px) {
+    width: 200px;
+  }
```

Comment:
I would put the width on all screen

Reply:
@roiLeo on larger ones , the search bar was looking really big tho

Reply:
Alright, then we could try to fix a width to input too as we don't need to search super long query